pkg/sql/types/types.go

```diff
@@ -1475,7 +1475,7 @@ var familyNames = map[Family]string{
 //
 // TODO(radu): investigate whether anything breaks if we use
 // enumvalue_customname and use String() instead.
-func (f Family) Name() string {
```

Comment:
Is this pattern of making fairly low-level packages depend on `redact` something we actually want to do?